### DISCUSSION TITLE
blog/2024/07/08/fun-with-positron/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#98] [Blog comments]
# blog/2024/07/08/fun-with-positron/

Combine the best of RStudio and Visual Studio Code in Posit’s new Positron IDE

https://www.andrewheiss.com/blog/2024/07/08/fun-with-positron/

**Comment by abduazizR** [comment]
Thanks a lot. I was actually looking forward for something like this from you for VScod, but this blog is much better than what I was waiting.

Many thanks

**Comment by mpaulacaldas** [comment]
I can even begin to explain how useful I found this post and how much time it has saved me. Thank you!!

**Comment by AlamFaraz** [comment]
Totally agree with this take. Remote SSH and container-based workflows are incredibly powerful, but the setup friction still feels higher than it needs to be—especially for people who just want to edit and run code on a remote machine without wrestling with configuration. If Open Remote-SSH became easier to install and supported seamless remote execution, it would be a huge productivity win. <a hrefv="https://pkr98game.net.pk">apkhup</a>

**Reply by andrewheiss** [reply]
Remote SSH is built in to Positron natively now: https://positron.posit.co/remote-ssh.html
